src/main/java/edu/hm/hafner/analysis/FullTextFingerprint.java

```diff
@@ -20,13 +20,13 @@
 
 /**
  * Creates a fingerprint of the specified issue using the source code at the affected line. The fingerprint is computed
- * using the 1:1 content of a small number of lines before and after the affected line (see {@link #LINES_LOOK_AHEAD}).
+ * using the 1:1 content of a small number of lines before and after the affected line (see {@link #linesLookAhead}).
  *
  * @author Ullrich Hafner
  */
 public class FullTextFingerprint {
     /** Number of lines before and after current line to consider. */
-    private static final int LINES_LOOK_AHEAD = 3;
```

Comment:
Do not remove this variable and rename it to `DEFAULT_ LINES_LOOK_AHEAD`. Then you can use it in the default constructor in line 41.